public/app/core/utils/fetch.ts

```diff
@@ -128,14 +128,18 @@ export async function parseResponseBody<T>(
         // TODO refactor this function to remove the type assertions
         return response.blob() as Promise<T>;
 
+      case 'body':
+        // Return the raw body -- useful for streaming
+        // TODO refactor this function to remove the type assertions
+        return Promise.resolve(response.body) as Promise<T>;
```

Comment:
@Clarity-89 I renamed "stream" to "body" because it is returning the response body... that we use for streaming 🤣